Cubical/Foundations/Equiv/PathSplit.agda

```diff
@@ -37,37 +38,36 @@ PathSplitEquiv A B = Σ[ f ∈ (A → B) ] isPathSplitEquiv f
 open isPathSplitEquiv
 
 idIsPathSplitEquiv : ∀ {ℓ} {A : Type ℓ} → isPathSplitEquiv (λ (x : A) → x)
-sec idIsPathSplitEquiv = (λ x → x) , (λ x → refl)
-secCong idIsPathSplitEquiv = λ x y → (λ p → p) , λ p _ → p
+sec idIsPathSplitEquiv     = (λ x → x) , (λ _ → refl)
+secCong idIsPathSplitEquiv = λ _ _ → (λ p → p) , λ p _ → p
 
 module _ {ℓ ℓ'} {A : Type ℓ} {B : Type ℓ'} where
+
+  open Iso
+
+  toIso : (f : A → B) → isPathSplitEquiv f → Iso A B
+  fun (toIso f _) = f
+  inv (toIso _ p) = p .sec .fst
+  rightInv (toIso _ p)  = p .sec .snd
+  leftInv (toIso f p) x = p .secCong (p .sec .fst (f x)) x .fst (p .sec .snd (f x))
+
   toIsEquiv : (f : A → B) → isPathSplitEquiv f → isEquiv f
-  toIsEquiv f record { sec = sec ; secCong = secCong } =
-    (isoToEquiv (iso f (fst sec) (snd sec) (λ x → (secCong (fst sec (f x)) x).fst (snd sec (f x))))) .snd
+  toIsEquiv f p = isoToIsEquiv (toIso f p)
 
   sectionOfEquiv' : (f : A → B) → isEquiv f → B → A
-  sectionOfEquiv' f isEqv x =
-    let all-fibers-contractible = isEqv .equiv-proof
-    in  all-fibers-contractible x .fst .fst
+  sectionOfEquiv' f isEqv x = isEqv .equiv-proof x .fst .fst
 
   isSec : (f : A → B) → (pf : isEquiv f) → section f (sectionOfEquiv' f pf)
-  isSec f isEqv x =
-    let all-fibers-contractible = isEqv .equiv-proof
-    in  all-fibers-contractible x .fst .snd
+  isSec f isEqv x = isEqv .equiv-proof x .fst .snd
 
   sectionOfEquiv : (f : A → B) → isEquiv f → hasSection f
   sectionOfEquiv f e = sectionOfEquiv' f e , isSec f e
 
 module _ {ℓ} {A B : Type ℓ} where
-  abstract
-    fromIsEquiv : (f : A → B) → isEquiv f → isPathSplitEquiv f
-    sec (fromIsEquiv f pf) = sectionOfEquiv' f pf , isSec f pf
-    secCong (fromIsEquiv f pf) x y = sectionOfEquiv cong-f eq-cong
-            where
-            cong-f : x ≡ y → f x ≡ f y
-            cong-f = λ (p : x ≡ y) → cong f p
-            eq-cong : isEquiv cong-f
-            eq-cong = isEquivCong (f , pf)
+
+  fromIsEquiv : (f : A → B) → isEquiv f → isPathSplitEquiv f
```

Comment:
@felixwellen : I managed to get rid of all the abstract's in this file without any real slow down in typechecking by using more copattern-matching and changing the proof of isEquivCong